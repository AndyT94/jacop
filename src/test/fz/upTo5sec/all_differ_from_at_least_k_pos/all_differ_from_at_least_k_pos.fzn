var 2..2: c:: output_var;
array [1..12] of var int: x:: output_array([1..3,1..4]) = [2,5,2,0,3,6,2,1,3,6,1,0];
array [1..12] of var int: X_INTRODUCED_12 ::var_is_introduced  = [2,5,2,0,3,6,2,1,3,6,1,0];
solve :: int_search(X_INTRODUCED_12,first_fail,indomain_min,complete) satisfy;
