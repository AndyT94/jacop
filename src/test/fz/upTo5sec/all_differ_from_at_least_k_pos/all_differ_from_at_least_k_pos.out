c = 2;
x = array2d(1..3, 1..4, [2, 5, 2, 0, 3, 6, 2, 1, 3, 6, 1, 0]);
----------